src/containers/Header/Header.scss

```diff
@@ -8,8 +8,17 @@
 
     padding: 0 20px 0 12px;
 
+    color: var(--g-color-text-primary);
     border-bottom: 1px solid var(--g-color-line-generic);
 
+    a {
```

Comment:
Lets not define styles for html-tags, it's bad practice, that may lead to collisions and unwanted side-effects. Please use custom classNames.